### PR TITLE
Add Valentin and Elliot as code owners for LLVM_jll and several related JLLs

### PR DESCRIPTION
**Pull request by DilumAluthge**
This will ensure that GitHub sends them a notification for any PRs that modify those JLLs.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,3 +4,12 @@ CODEOWNERS @DilumAluthge
 
 .ci/parse_env.jl @ericphanson
 .github/workflows/feed.yml @DilumAluthge @ericphanson
+
+C/Clang_assert_jll/ @vchuravy @staticfloat
+C/Clang_jll/ @vchuravy @staticfloat
+L/LLVMExtra_jll/ @vchuravy @staticfloat
```
Comment (vchuravy) [inline .github/CODEOWNERS:10]:
```suggestion
L/LLVMExtra_jll/ @vchuravy @maleadt 
```

Don't know if @maleadt has the merge bit here.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -4,3 +4,12 @@ CODEOWNERS @DilumAluthge
...
+L/LLVMLibUnwind_jll/ @vchuravy @staticfloat
+L/LLVM_assert_jll/ @vchuravy @staticfloat
+L/LLVM_full_assert_jll/ @vchuravy @staticfloat
+L/LLVM_full_jll/ @vchuravy @staticfloat
+L/LLVM_jll/ @vchuravy @staticfloat
```
Comment (vchuravy) [inline .github/CODEOWNERS:15]:
```suggestion
L/LLVM_jll/ @vchuravy @staticfloat
M/MLIR_jll/ @vchuravy @staticfloat
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (DilumAluthge) [reply .github/CODEOWNERS:10]: I just checked, and he does.

**Comment by vchuravy** [comment]
Thanks!

**Pull request merged**